### PR TITLE
[IMP] document_page_approval_levels: modify access control list and add record rules

### PR DESCRIPTION
**Pull request by glizek**
[TA#7688](https://www.vauxoo.com/web#id=7688&view_type=form&model=project.task&menu_id=136&action=138)

### CODE REVIEW

**Comment by sabrinagnu** [comment]
Functional test 👍  @JulioSerna  may you please gave me your technical review?

File: document_page_approval_levels/security/res_groups.xml [hunk 1]
```diff
@@ -14,5 +14,16 @@
             <field name="implied_ids" eval="[(4, ref('group_document_approver'))]"/>
         </record>
 
+       <record id="group_document_user_rule" model="ir.rule">
+            <field name="name">Can only delete your document</field>
+            <field name="model_id" ref="document_page.model_document_page"/>
+            <field name="domain_force">['|', ('create_uid', '=', user.id), ('create_uid', '=', False)]</field>
```
Comment (JulioSerna) [inline document_page_approval_levels/security/res_groups.xml]: in what case `create_uid` is False?

File: document_page_approval_levels/security/res_groups.xml [hunk 2]
```diff
@@ -14,5 +14,16 @@
...
+            <field name="groups" eval="[(4, ref('base.group_document_user'))]" />
+            <field name="perm_read" eval="0"/>
```
Comment (JulioSerna) [inline document_page_approval_levels/security/res_groups.xml]: you need make first the read in document before make unlink

what this rule is working?

**Review by JulioSerna — APPROVED** [review]

**Comment by JulioSerna** [comment]
@glizek 

maybe you  should add one tests to validate that other user different from the creator can not deleted document

**Pull request merged**